bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -187,6 +187,16 @@ impl Room {
         })
     }
 
+    pub fn member(&self, user_id: String) -> Result<Arc<RoomMember>, ClientError> {
+        let room = self.room.clone();
+        let user_id = user_id;
+        RUNTIME.block_on(async move {
+            let user_id = <&UserId>::try_from(&*user_id).context("Invalid user id.")?;
```

Comment:
[`UserId::parse()`](https://docs.rs/ruma-identifiers/latest/ruma_identifiers/struct.UserId.html#method.parse) is a bit more pleasant to look at. Let's use it.

Reply:
Well, that one allocates which `<&UserId>::try_from` doesn't ^^
But maybe it's not all that relevant.